game/core/src/main/java/com/gg/game/view/Assets.java

```diff
@@ -11,20 +11,20 @@
  */
 class Assets {
 
-    // A Map to store which image belongs to which object
-    private final Map<Object, Texture> objectTextureMap = new HashMap<>();
+    // A Map to store which image belongs to which class
+    private final Map<Class, Texture> objectTextureMap = new HashMap<>();
 
     /**
      * Binds objects to textures using a map.
      *
-     * @param object        the object to bind to texture.
+     * @param clazz the class to bind to texture.
```

Comment:
```suggestion
     * @param clazz the class to bind to texture.
```
Is it standard to use clazz like this? Or is it just in tutorials where `foo()` and `bar()` etc are used?